sys/include/net/gnrc/priority_pktqueue.h

```diff
@@ -1,22 +1,23 @@
 /*
  * Copyright (C) 2015 Daniel Krebs
+ *               2016 INRIA
  *
  * This file is subject to the terms and conditions of the GNU Lesser
  * General Public License v2.1. See the file LICENSE in the top level
  * directory for more details.
  */
 
 /**
- * @{
- * @defgroup    net_gnrc_priority_pktqueue Priority packet queue
+ * @defgroup    net_gnrc_priority_pktqueue Priority packet queue for GNRC
```

Comment:
Not really necessary since in the HTML view this is ordered under GNRC and in the file view this is ordered under the gnrc headers.